ui/r4r/src/main/kotlin/de/cyface/app/r4r/capturing/CapturingViewModel.kt

```diff
@@ -138,7 +139,9 @@ class CapturingViewModel(
      * the updated track is requested. This is `null` if there is no unfinished measurement.
      */
     fun setTracks(tracks: List<Track>?) {
-        _tracks.postValue(if (tracks == null) null else if (tracks.isEmpty()) arrayListOf() else tracks as ArrayList<Track>)
+        _tracks.postValue(
+            if (tracks == null) null else if (tracks.isEmpty()) arrayListOf() else tracks as ArrayList<Track>
```

Comment:
emptyList? Why is it necessary to explicitly use an ArrayList implementation here?

Reply:
https://github.com/cyface-de/android-app/pull/98/commits/290da5435e0399de73b504d1bee23dc9b0a1d259

https://github.com/cyface-de/android-app/pull/98/commits/c37a766d33b747848d8c69cf2d81eae4a6953b95